Gemfile

```diff
@@ -152,7 +152,7 @@ gem 'swagger-blocks'
 # POSIX systems should have this already, so we're not going to bring it in on other platforms
 gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 gem 'utf8-cleaner'
-gem 'vets_json_schema', git: 'https://github.com/department-of-veterans-affairs/vets-json-schema', branch: 'master'
```

Comment:
Is this for testing purposes only?  Normally we would leave this pointing to master?